src/lib/structures/Command.ts

```diff
@@ -43,6 +43,13 @@ export abstract class Command<T = Args> extends AliasPiece {
 	 */
 	public strategy: Lexure.UnorderedStrategy;
 
+	/**
+	 * If the client's options has set typing as true, it can be overriden for a specific command by setting it to false in its options.
+	 * If the client's options has set typing as false, the typing property in the command will be ignored.
```

Comment:
```suggestion
	 * If {@link SapphireClient.typing} is true, it can be overridden for a specific command using this property.
	 * Otherwise, this property will be ignored.
```

Reply:
This could be simplified to something along the lines of "Allows overriding the default {@link SapphireClient.typing} setting for this command.", which sounds better in my opinion.

Reply:
It is a little misleading that way, because it doesn't override the default setting on all cases, but only when `SapphireClient.typing` is `true`. `CommandOptions.typing = true` never does anything.